aws-lc-sys/CMakeLists.txt

```diff
@@ -10,43 +10,58 @@ enable_language(C)
 
 add_subdirectory(aws-lc aws-lc EXCLUDE_FROM_ALL)
 
-add_definitions(-DAWS_LC_RUST_INCLUDE_SSL)
+if (BUILD_LIBSSL)
+    add_definitions(-DAWS_LC_RUST_INCLUDE_SSL)
+endif()
 add_library(rust_wrapper STATIC rust_wrapper.c)
 target_include_directories(rust_wrapper PRIVATE include)
-target_link_libraries(rust_wrapper PUBLIC crypto ssl)
+
+if (BUILD_LIBSSL)
+    target_link_libraries(rust_wrapper PUBLIC crypto ssl)
+else()
+    target_link_libraries(rust_wrapper PUBLIC crypto)
+endif()
```

Comment:
Simplify to:
```cmake
target_link_libraries(rust_wrapper PUBLIC crypto)
if (BUILD_LIBSSL)
    target_link_libraries(rust_wrapper PUBLIC ssl)
endif()
```
> Repeated calls for the same <target> append items in the order called.

https://cmake.org/cmake/help/latest/command/target_link_libraries.html#overview

Reply:
Fixed in next commit.